package.json

```diff
@@ -14,6 +14,7 @@
 		"express-favicon": "^2.0.4",
 		"firebase": "^10.12.4",
 		"firebase-admin": "^12.3.0",
+		"google-auth-library": "^9.14.1",
 		"jsonwebtoken": "^9.0.2",
 		"jwt-decode": "^4.0.0",
 		"mongoose": "^7.0.1",
```

Comment:
Do we need to install "google-auth-library" package on our local computer too?

Reply:
yes just do the npm install